lib/worker.js

```diff
@@ -93,6 +95,22 @@ async function getLocalLinter(basedir) {
   });
 }
 
+async function getNodePrefixPath() {
+  const npmCommand = process.platform === 'win32' ? 'npm.cmd' : 'npm';
+  let nodePrefixPath;
+  try {
+    nodePrefixPath = ChildProcess.spawnSync(npmCommand, ['get', 'prefix'], {
```

Comment:
We shouldn't use the `*Sync` versions of system commands, this takes processing time away from other atom packages due to us blocking on io

Reply:
Whoops, my mistake! In my haste to port over `linter-eslint`'s implementation, I forgot to adjust it for the fact that the rest of the code here is all nice and async'd. Didn't catch that they were using `spawnSync()`, although in retrospect it should have been obvious. xD I'll get right on that. 😅